app/models/like.rb

```diff
@@ -0,0 +1,16 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  after_create :increment_counter
+  after_destroy :decrement_counter
+
+  private
+
+  def increment_counter
+    post.increment!(:likes_counter)
+  end
+
+  def decrement_counter
+    post.decrement!(:likes_counter)
+  end
```

Comment:
@BiancaMe Great work adding the increment and decrement counter methods, I love this creativity especially the use of [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_create and after_destroy methods, keep up the great work